src/Utilities/PointerVector.hpp

```diff
@@ -1401,6 +1401,37 @@ PointerVector<Type, AF, PF, TF, ExprResultType>::divAssign(
 }
 /// \endcond
 
+template <typename Type, bool AF, bool PF, bool TF, typename ExprResultType>
```

Comment:
Could you please put this in a macro so we can generate the code for `double` and `std::complex<double>`?